flytekit/core/array_node_map_task.py

```diff
@@ -29,6 +29,7 @@
 from flytekit.tools.module_loader import load_object_from_module
 from flytekit.types.pickle import pickle
 from flytekit.types.pickle.pickle import FlytePickleTransformer
+from flytekit.utils.asyn import loop_manager
```

Comment:
Can this be imported as `run_sync` and then use that? (I prefer to hide the loop manager as much as possible)

Reply:
i guess so.  i was hoping to one day figure out type hinting, which doesn't work today in either case, but i think will never work with the run sync way.  I tried briefly getting functool wraps to work but alas to no avail.

Reply:
ohh it was missing a paramspec.  it works now. and changed promise.py and base_task.py to just use the run_sync.